### PR TITLE
Little bit improving code style and php docs

### PR DESCRIPTION
**Pull request by fr05t1k**
code style
phpdoc fix
isset to array_key_exists

for issue https://github.com/grpc/grpc/issues/7840

### CODE REVIEW

File: src/php/lib/Grpc/AbstractCall.php [hunk 1]
```diff
@@ -68,25 +77,27 @@ public function __construct(Channel $channel,
         $this->deserialize = $deserialize;
         $this->metadata = null;
         $this->trailing_metadata = null;
-        if (isset($options['call_credentials_callback']) &&
+        if (array_key_exists('call_credentials_callback', $options) &&
```
Comment (thinkerou) [inline src/php/lib/Grpc/AbstractCall.php]: Why use `array_key_exists`?

Comment (fr05t1k) [inline src/php/lib/Grpc/AbstractCall.php]: @thinkerou we need to check index in array, not a null value. I think it's more readable and clearly. Isn't it?

Comment (thinkerou) [inline src/php/lib/Grpc/AbstractCall.php]: Eh, right.

**Comment by stanley-cheung** [comment]
Thanks for the contribution.

**Pull request merged**